tests/integration/auth_test.go

```diff
@@ -3,58 +3,47 @@ package tests
 import (
 	"net/http"
 	"net/http/httptest"
-	"testing"
 
-	"github.com/Real-Dev-Squad/tiny-site-backend/routes"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestLogout(t *testing.T) {
-	router := routes.SetupV1Routes(db)
+func (suite *AppTestSuite) TestLogout() {
+    router := gin.Default()
+    router.GET("/v1/auth/logout", func(ctx *gin.Context) {
+        ctx.Redirect(http.StatusFound, "/home")
+    })
```

Comment:
when logout, why are we redirecting it to home?
And does the actual logout code gets called?

Reply:
actual code is this 
![image](https://github.com/Real-Dev-Squad/tiny-site-backend/assets/111434418/341aff58-4ab8-47e2-b447-5acac037e95f)

Reply:
so here you are clearing the cookie right, but in test its only redirect

Reply:
for auth test I'll take care in issue #55 as need to add controller file for that